wagtail_localize/views/submit_translations.py

```diff
@@ -222,7 +226,10 @@ def _walk(current_page):
         # TODO: Button that links to page in translations report when we have it
         messages.success(self.request, self.get_success_message(locales))
 
-        return redirect(self.get_success_url() or self.get_default_success_url())
+        return redirect(
+            self.get_success_url()
+            or self.get_default_success_url(single_translated_object)
```

Comment:
I've just tested this locally.
This bit is potentially confusing as the only way you can get to the translation edit page as it stands is if you tree syncing enabled and you click "translate this page" on the alias edit view

When you use the "more" actions from a page listing, `self.get_success_url()` is non-empty because there's a `next` query param with a value going back to the listing page (e.g. `admin/localize/submit/page/3/?next=%2Fadmin%2Fpages%2F60%2F`)

so perhaps move the single_translated_object to `get_success_url` ?

Reply:
alternatively, remove the next_url handling from https://github.com/wagtail/wagtail-localize/blob/main/wagtail_localize/wagtail_hooks.py#L153-L154 🤔

Reply:
I went with your second suggestion. I think it makes sense that we remove the "next" param from the "Translate this page" button as the view doesn't need it to know where to redirect the editor next.

This also means that we can keep the "next URL routing" feature if we needed it at some point.